src/sentry/rules/conditions/event_frequency.py

```diff
@@ -421,13 +421,18 @@ def batch_query_hook(
             "id", "type", "project__organization_id"
         )
         error_issue_ids, generic_issue_ids = self.get_error_and_generic_group_ids(groups)
+        organization_id = None
+        if groups:
+            group = groups[0]
+            if len(group) == 3:
+                organization_id = group[3]
```

Comment:
Should this be 2? (len 3 is 👍 -- good ol' off by 1s)
```suggestion
                organization_id = group[2]
```

Reply:
lol oops yes